### PR TITLE
feat: Alert & button

### PR DESCRIPTION
**Pull request by Creator-singh**

### CODE REVIEW

**Review by Aditi0505 — APPROVED** [review]
LGTM

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    justify-content: center;
+    background-color: var(--bg-warning);
+}
+
+.error-alert{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--bg-danger);
+}
```
Comment (rohanmathur91) [inline components/alert/alert.css:53]: You can have a utility class for flex.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+.info-alert{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--bg-info);
```
Comment (rohanmathur91) [inline components/alert/alert.css]: You can have a utility class for flex.

**Review by rohanmathur91 — APPROVED** [review]
Add one comment, rest looks good.

File: components/alert/alert.css (hunk 2 above)
Comment (Creator-singh) [reply components/alert/alert.css]: okay will do it

**Pull request merged**